lib/include/ert/ecl/ecl_file_kw.h

```diff
@@ -56,6 +56,9 @@ typedef struct inv_map_struct inv_map_type;
   void                ecl_file_kw_fwrite( const ecl_file_kw_type * file_kw , FILE * stream );
   ecl_file_kw_type ** ecl_file_kw_fread_alloc_multiple( FILE * stream , int num);
   ecl_file_kw_type *  ecl_file_kw_fread_alloc( FILE * stream );
+
+  void                ecl_file_kw_increment_ref_count(ecl_file_kw_type * file_kw);
+  int                 ecl_file_kw_get_ref_count(ecl_file_kw_type * file_kw);
```

Comment:
Do not want these out of `ecl_file_kw` To be able to do that the implementation of `ecl_file_transaction` must be in the `ecl_file_view.c` file.